### PR TITLE
chore: add to default config

### PR DESCRIPTION
**Pull request by lesam**
###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [ ] ~~CHANGELOG.md updated~~ - No changelog for chores

### CODE REVIEW

**Comment by lesam** [comment]
This should be reviewed by the docs team as well.

**Review by docmerlin — APPROVED** [review]
100%

File: etc/kapacitor/kapacitor.conf [hunk 1]
```diff
@@ -99,6 +121,21 @@ default-retention-policy = ""
   # The message of the alert. INTERVAL will be replaced by the interval.
   message = "{{ .ID }} is {{ if eq .Level \"OK\" }}alive{{ else }}dead{{ end }}: {{ index .Fields \"collected\" | printf \"%0.3f\" }} points/INTERVAL."
 
+[fluxtask]
+  # Configure flux tasks for kapacitor
+  enabled = false
+  # The influxdb instance to store historical task run data (the 'name' from the influxdb config section)
```
Comment (kelseiv) [inline etc/kapacitor/kapacitor.conf:127]: @lesam, default influxdb connection (if multiple)?

File: etc/kapacitor/kapacitor.conf [hunk 2]
```diff
@@ -99,6 +121,21 @@ default-retention-policy = ""
...
+  # Not recommended: use "none" to turn off historical task run data storage.
+  task-run-influxdb = "localhost"
+  # The bucket (for InfluxDB 1.x, use "db/rp" format) to store historical task run data.
```
Comment (kelseiv) [inline etc/kapacitor/kapacitor.conf:130]: may be helpful to provide exact format for db/rp example

File: etc/kapacitor/kapacitor.conf [hunk 3]
```diff
@@ -432,11 +469,16 @@ default-retention-policy = ""
...
   workspace = ""
 
-  # The Slack webhook URL, can be obtained by adding
-  # an Incoming Webhook integration.
-  # Visit https://slack.com/services/new/incoming-webhook
-  # to add new webhook for Kapacitor.
+  # The Slack webhook URL
+  # For new-style slack apps, use "https://slack.com/api/chat.postMessage"
+  # For legacy Incoming Webhooks, visit https://slack.com/services/new/incoming-webhook
+  # to add new webhook for Kapacitor and use that URL.
```
Comment (kelseiv) [inline etc/kapacitor/kapacitor.conf:475]:
```suggestion
  # to add a new webhook for Kapacitor and use that URL.
```

File: etc/kapacitor/kapacitor.conf [hunk 4]
```diff
@@ -432,11 +469,16 @@ default-retention-policy = ""
...
   url = ""
+  # The Slack OAuth token
+  # For new-style slack apps, visit https://api.slack.com/apps -> your app -> 'OAuth & Permissions' to find
+  # the token. Ensure your app has 'chat:write' and 'chat:write.public' permissions.
+  # For legacy Incoming Webhooks this should be left empty.
```
Comment (kelseiv) [inline etc/kapacitor/kapacitor.conf:480]:
```suggestion
  # For legacy Incoming Webhooks, leave empty.
```

**Review by kelseiv — APPROVED** [review]
Looks good

File: etc/kapacitor/kapacitor.conf (hunk 1 above)
Comment (lesam) [reply etc/kapacitor/kapacitor.conf:127]: Yes

**Pull request merged**